docs/installation/autoware/source-installation.md

```diff
@@ -65,6 +65,10 @@ sudo apt-get -y install git
 
 ## How to set up a workspace
 
+!!! info [Using Autoware Build GUI](#using-autoware-build-gui)
```

Comment:
`!!! info` should either be alone or like in https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title

> ```
> !!! note "Phasellus posuere in sem ut cursus"
>
>   Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod
>   nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor
>   massa, nec semper lorem quam in massa.
> ```

Right now it looks broken:
- https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#how-to-set-up-a-workspace

![Screenshot from 2024-01-29 17-34-15](https://github.com/autowarefoundation/autoware-documentation/assets/10751153/1478ae7d-6e5d-4c09-b92a-6af04872138a)

@KhalilSelyan could you create a mini PR to fix this?